frontend/pulse3d/src/components/interactiveAnalysis/InteractiveAnalysisModal.js

```diff
@@ -121,9 +121,9 @@ const constantModalLabels = {
     buttons: ["Close"],
   },
   duplicate: {
-    header: "Before Running Analysis!",
-    messages: ["Check no double peaks or valleys exist.", "They are shown as red triangles above."],
-    buttons: ["Close"],
+    header: "Warning!",
+    messages: ["Duplicate peaks and valleys detected."],
```

Comment:
This message is a little ambiguous, someone may think it means that both duplicate peaks and duplicate valleys exist. Could just change it to `"Duplicate peaks and/or valleys detected."`